templates/form-fields/complex-field.php

```diff
@@ -94,8 +93,7 @@
 			$value = isset( $values[ $index ][ $key ] ) ? $values[ $index ][ $key ] : '';
 
 			if ( $index > 0 && ! $clone_row ) {
-				$key .= '_' . $index;
-
+				//$key .= '_' . $index;
```

Comment:
Is this not needed?

Reply:
@polevaultweb 
Yes Iain.
Adding the _ + index will cloned the field key in the repeater array but it was not actually used in the front-end.